apps/family-public-law/fpl-cron-ccd-data-migration-tool/prod/00.yaml

```diff
@@ -6,9 +6,9 @@ spec:
   releaseName: fpl-cron-ccd-data-migration-tool
   values:
     job:
-      schedule: "45 19 19 10 *"
+      schedule: "00 12 26 10 *"
       environment:
         ENABLED: true
-        MIGRATION_ID: DFPL-AM
+        MIGRATION_ID: DFPL-1813
         IDAM_API_URL: https://idam-api.platform.hmcts.net
         USE_CASE_ID_MAPPING: false
```

Comment:
Just wondering if we need to set USE_CASE_ID_MAPPING to true if we're specifying the case id to run this for?

Reply:
Pushed a commit to change the value to true